src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.UnitTestingNormalPriorityProcessor.cs

```diff
@@ -352,6 +361,7 @@ private async Task ProcessDocumentAsync(ImmutableArray<IUnitTestingIncrementalAn
                                         await ProcessOpenDocumentIfNeededAsync(analyzers, workItem, textDocument, isOpen, cancellationToken).ConfigureAwait(false);
                                         await ProcessCloseDocumentIfNeededAsync(analyzers, workItem, textDocument, isOpen, cancellationToken).ConfigureAwait(false);
                                     }
+#endif
```

Comment:
note: as unit testing doesn't care abotu open/close events, the next pr removes those events from teh ones that VS will send to OOP.

Reply:
I'm curious whether this would have triggered analysis that would have called AnalyzeDocument() on the UT analyzer when files are opened (and whether commenting it out can cause any change in semantics esp. if we want to keep the highPriorityForActiveFiles flag above). I am assuming not - but the 'if needed' in the name of the above API prompted this question...

Reply:
so all ProcessOpenDocumentIfNeededAsync does it run the DocumentOpenAsync method on all registered analyzers.  so for UnitTesting that was a no-op.  So these two calls were no ops.  so i if'def'ed it out. :)